app/models/accessionable/base.rb

```diff
@@ -79,16 +79,82 @@ def label_scope
   class Tag # rubocop:todo Style/Documentation
     attr_reader :value
 
+    # Value serialization for accessioning XML generation
+    # It will return the same value without any changes and it applies
+    # to all tags
+    class FieldSerializer
+      NOT_COLLECTED = 'not collected'
+      NOT_PROVIDED = 'not provided'
+      RESTRICTED_ACCESS = 'restricted access'
+
+      OTHER_DEFAULT_SETTINGS = [NOT_COLLECTED, NOT_PROVIDED, RESTRICTED_ACCESS].freeze
```

Comment:
'not applicable' is also given in config/locales/metadata/en.yml  country_of_origin help